<launch>

  <arg name="namespace_prefix" default="irb120_eih_calib" />
  <arg name="eye_on_hand" default="true" />
  <arg if="$(arg eye_on_hand)" name="namespace" value="$(arg namespace_prefix)_eye_on_hand" />
  <arg unless="$(arg eye_on_hand)" name="namespace" value="$(arg namespace_prefix)_eye_on_base" />

  <arg name="robot_base_frame" value="base_link" />
  <arg name="robot_effector_frame" value="tool0" />
  <arg name="tracking_base_frame" value="camera_color_optical_frame" />
  <arg name="tracking_marker_frame" value="tag_0" />

  <!-- publish the hand-eye calibration computed with easy_handeye -->
  <include file="$(find easy_handeye)/launch/publish.launch" >
    <arg name="eye_on_hand" value="true" />
    <arg name="namespace_prefix" value="$(arg namespace_prefix)" />
  </include>

</launch>